Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -16208,8 +16206,6 @@ final class WorkspaceTests: XCTestCase {
     }
 
     func testInvalidTraitConfiguration_ForRootPackage() async throws {
-        try XCTSkipOnWindows(because: #"\tmp\ws doesn't exist in file system"#)
```

Comment:
Why is this removed? How could this pass on Windows?

Reply:
I suspect there were two PR in flight at the same time that may have caused a "merge race"

https://github.com/swiftlang/swift-package-manager/pull/8457 and https://github.com/swiftlang/swift-package-manager/pull/8370

Reply:
/tmp/ws isn't a valid absolute path on Windows, so I'm surprised this passes, but if it does, it does. That's more of an issue with AbsolutePath.

Reply:
The test is using an `InMemoryFileSystem()`, and the `AbsolutePath("/tmp/ws/")`is.  The although `AbsolutePath("/tmp/fs")` is not a "real" Windows absolute path, there lies an issue with Swift Tools Support Core's `AbsoluePath` since the `Basics.AbsolutePath` is a wrapper around STSC Absolute Path.

In the long term, I would like to convert SwiftPM `AbsolutePath` and `RelativePath` usages to a more modern, and platform agnostic, Path library, perhaps https://github.com/apple/swift-system/blob/main/Sources/System/FilePath/FilePath.swift, or maybe the sane path library use by https://github.com/swiftang/swift-build